en/models/virtual-fields.rst

```diff
@@ -120,6 +120,76 @@ best to define the virtualFields in your model's constructor::
 This will allow your virtualFields to work for any alias you give a
 model.
 
+Pagination and Virtual fields set in controller with JOINS
+=============================================================
+The following example allows to have a counter of a hasMany association and allows to use without problem
```

Comment:
No problem, my English is very bad.
I helped 100% with Google Translate.
With that you understand my text and your correction correct that concept to a bleated English, it is perfect for me. 👍 
Thank you.